scipy/stats/_distribution_infrastructure.py

```diff
@@ -5219,3 +3582,288 @@ def __rtruediv__(self, other):
 
     def __neg__(self):
         return self * -1
+
+
+class Mixture(_ProbabilityDistribution):
+    r""" Class that represents a mixture distribution.
+
+    A mixture distribution is the distribution of a random variable
+    defined in the following way: first, a random variable is selected
+    from `components` according to the probabilities given by `weights`, then
+    the random variable is realized[1]_.
+
+    Parameters
+    ----------
+    components : sequence of `ContinuousDistribution`
+        The underlying instances of `ContinuousDistribution`.
+        All must have scalar shape parameters.
```

Comment:
> Do you mean all elements in components must be scalar valued? (e.g. not using “vectorized” parameters?)

Yes, IIUC. Elements in `components` are instances of `ContinuousDistribution`, so while I think I understand what you mean by them being "scalar valued", I would not write this in the documentation because they are not "scalar values" in the sense of floats and ints. If the `ContinuousDistribution` has parameters, those parameters must be scalar-valued *at the moment* for use in `Mixture`. (`ContinuousDistribution` itself works with "vectorized" N-d shape parameters, and all the underlying functions for integration, root-finding, and minimization work with N-d arrays natively; I just ran into some shape issues I'd rather sort out in a separate PR.)

> And I guess “shape parameters” refers to shape/loc/scale parameters? 

Yes. The `pdf` method of the underlying objects needs to return a scalar if the argument is a scalar.

I thought about making the `_shape` attribute of instances of `ContinuousDistribution` public; this would simplify exposition in such cases.

Reply:
Thanks for the explanations. It’s clear now. Exposing the `shape` attribute of a `ContinuousDistribution` instance sounds a great idea for future consideration!